src/modules/storagefacility/TableComponent.js

```diff
@@ -102,6 +104,70 @@ export class StorageUnitsContainer extends React.Component {
     });
   }
 
+  componentDidMount() {
+    this.scanner = subscribeToScanner((barCode) => {
+      this.props.clear();
+      const museumId = this.props.appSession.getMuseumId();
+      const collectionId = this.props.appSession.getCollectionId();
+      const token = this.props.appSession.getAccessToken();
+      if (barCode.uuid) {
+        this.props.findByUUID({uuid: barCode.code, museumId, token}).do((response) => {
+          if (!response) {
+            this.props.emitError({message: I18n.t('musit.errorMainMessages.scanner.noMatchingNode', {uuid: barCode.code})});
+          } else {
+            const isMoveHistoryActive = document.getElementsByClassName('moveHistory').length > 0;
+            const isMoveDialogActive = document.getElementsByClassName('moveDialog').length > 0;
```

Comment:
This feels like a code smell! Is it easy to send in this state so we don't need to interact with the DOM.